jxl/src/enc/numpy.rs

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) the JPEG XL Project Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+#![cfg(feature = "debug_tools")]
```

Comment:
(you can put this on the `pub mod numpy`)

Reply:
done